schema/query-log-schema-2g.json

```diff
@@ -0,0 +1,167 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "definitions": {
+
+    "unit": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
+            "second",
+            "minute",
+            "hour",
+            "day",
+            "week",
+            "bit",
+            "byte",
+            "rotation",
+            "degree",
+            "radian",
+            "celsius",
+            "fahrenheit",
+            "kelvin"
+          ]
+        },
+        "scale": {
+          "type": "string",
+          "enum": [
+            "yocto",
+            "zepto",
+            "atto",
+            "femto",
+            "pico",
+            "nano",
+            "micro",
+            "milli",
+            "centi",
+            "deci",
+            "deca",
+            "hecto",
+            "kilo",
+            "mega",
+            "giga",
+            "tera",
+            "peta",
+            "exa",
+            "zetta",
+            "yotta",
+            "kibi",
+            "mebi",
+            "gibi",
+            "tebi",
+            "pebi",
+            "exbi",
+            "zebi",
+            "yobi",
+          ]
+        }
+      }
+    },
+    "sample": {
+      "type": "object",
+      "properties": {
+        "unitNumerators": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/unit"
+          }
+        },
+        "unitDenominators": {
```

Comment:
I don't think your example matches your spec. Please verify. Specifically numerator and denominator should be nested under unit.